platform/infra/lib/infra/vpc.ts

```diff
@@ -9,8 +9,41 @@ export default class VpcStack extends Stack {
     constructor(scope: Construct, id: string, props: StackProps) {
         super(scope, id, props);
         this.vpc = new Vpc(this, "appmod-vpc", {
-            vpcName: "appmod-vpc"
+            vpcName: "appmod-vpc",
+            maxAzs: 3,
+            ipAddresses: IpAddresses.cidr('10.0.0.0/16'),
+            subnetConfiguration: [
+                {
+                  cidrMask: 19,
+                  name: 'public-subnet',
+                  subnetType: SubnetType.PUBLIC,
+                },
+                {
+                  cidrMask: 19,
+                  name: 'private-egress-subnet',
+                  subnetType: SubnetType.PRIVATE_WITH_EGRESS,
+                }
+            ]
         });
-    }
 
+        const publicSubnets = this.vpc.selectSubnets({
+            subnetType: SubnetType.PUBLIC
+        }).subnets;
+
+        const privateSubnets = this.vpc.selectSubnets({
+            subnetType: SubnetType.PRIVATE_WITH_EGRESS
+        }).subnets;
+
+        privateSubnets.forEach((subnet) => {
```

Comment:
let's do it for public subnets as well. I assume a public facing ALB will be created in the public subnet.

Reply:
Resolved in [8d2664b](https://github.com/shapirov103/appmod-blueprints/pull/15/commits/8d2664b939340acdb92e6216d5d3158388332f85)